ui/src/app/views/recent-work/component.ts

```diff
@@ -22,27 +22,51 @@ import {convertToResources, ResourceType} from 'app/utils/resourceActions';
 
 export class RecentWorkComponent implements OnInit {
   resourceList: RecentResource[];
+  resourcesLoading: boolean;
   fullList: RecentResource[] = [];
+  @Input('headerText')
+  headerText = 'Your Recent Work';
   @Input('workspace')
   workspace: Workspace;
   accessLevel: WorkspaceAccessLevel;
   startIndex = 0;
   cssClass: string;
-  resourcesLoading = false;
   constructor(
     private userMetricsService: UserMetricsService,
     private workspacesService: WorkspacesService,
     private cohortsService: CohortsService,
     private route: ActivatedRoute,
   ) {}
   index: Number;
-  size = 3;
+  @ViewChild('recentWork')
+  eMainFrame: ElementRef;
```

Comment:
Nit: What is `eMainFrame`? Can we call this recentWork or something?

Reply:
sure! it's basically just the whole component element